t/torchvision/torchvision_ubi_9.3.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : vision
+# Version           : v0.16.1
+# Source repo       : https://github.com/pytorch/vision.git
+# Tested on         : UBI:9.3
+# Language          : Python
+# Travis-Check      : True
+# Script License    : Apache License, Version 2.0
+# Maintainer        : Md. Shafi Hussain <Md.Shafi.Hussain@ibm.com>
+#
+# Disclaimer        : This script has been tested in root mode on given
+# ==========          platform using the mentioned version of the package.
+#                     It may not work as expected with newer versions of the
+#                     package and/or distribution. In such case, please
+#                     contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=vision
+PACKAGE_VERSION=${1:-v0.16.1}
+PACKAGE_URL=https://github.com/pytorch/vision.git
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+PYTHON_VER=${2:-3.9}
+PARALLEL=${PARALLEL:-$(nproc)}
+
+WORKDIR=$(pwd)
+
+dnf install -y https://mirror.stream.centos.org/9-stream/BaseOS/ppc64le/os/Packages/centos-gpg-keys-9.0-24.el9.noarch.rpm \
+            https://mirror.stream.centos.org/9-stream/BaseOS/`arch`/os/Packages/centos-stream-repos-9.0-24.el9.noarch.rpm \
+            https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+dnf config-manager --add-repo https://mirror.stream.centos.org/9-stream/AppStream/`arch`/os
+dnf config-manager --set-enabled crb
+dnf install -y git cmake ninja-build g++ rust cargo jq \
+            python${PYTHON_VER}-devel python${PYTHON_VER}-wheel python${PYTHON_VER}-pip python${PYTHON_VER}-setuptools \
+            libtiff-devel libjpeg-devel openjpeg2-devel zlib-devel \
+            libpng-devel freetype-devel lcms2-devel libwebp-devel tcl-devel tk-devel \
+            harfbuzz-devel fribidi-devel libraqm-devel libimagequant-devel libxcb-devel
+
+if ! command -v python; then
+    ln -s $(command -v python${PYTHON_VER}) /usr/bin/python
+fi
+if ! command -v pip; then
+    ln -s $(command -v pip${PYTHON_VER}) /usr/bin/pip
+fi
+
```

Comment:
No need to create symbolic link for `python`.
Your build script should only have the instructions about the package requirements. 
Installation of python, supporting different python versions etc, is already taken care in the wrapper script. So we don't need repetitive instructions.
For testing purpose you can create python venv in your ssh terminal and hit this shell script.

Reply:
@shubham-dayma-ibm Is the Travis CI using the wrapper script already?
If not then, I do need to install pip and create sym-link for `python`
The base image (UBI9:9.3) used by Travis has python3 installed but not aliased to python. It needs pip to be installed as well